metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/ElasticSearchService.java

```diff
@@ -144,7 +144,7 @@ public AutoCompleteResult autoComplete(@Nonnull String entityName, @Nonnull Stri
   @Override
   public Map<String, Long> aggregateByValue(@Nullable List<String> entityNames, @Nonnull String field,
       @Nullable Filter requestParams, int limit) {
-    log.debug("Aggregating by value: {}, field: {}, requestParams: {}, limit: {}", entityNames.toString(), field,
+    log.debug("Aggregating by value: {}, field: {}, requestParams: {}, limit: {}", entityNames != null ? entityNames.toString() : null, field,
```

Comment:
You don't need the toString call here at all, you can just do entityNames and it will auto resolve to the String form or null without risk of NPE